tests/func-tests/monitoring_test.go

```diff
@@ -46,8 +49,8 @@ var _ = Describe("[crit:high][vendor:cnv-qe@redhat.com][level:system]Monitoring"
 
 	var err error
 	var virtCli kubecli.KubevirtClient
-	var promClient promApiv1.API
 	var prometheusRule monitoringv1.PrometheusRule
+	var initialHealthMetricValue int
```

Comment:
you should set up the value of `initialHealthMetricValue` in the `BeforeEach`

Reply:
Done